app/mailers/publisher_mailer.rb

```diff
@@ -156,6 +156,114 @@ def unverified_domain_reached_threshold_internal(domain, email)
     )
   end
 
+  def channel_contested(channel)
+    @channel = channel
+    @email = channel.publisher.email
+
+    @transfer_url = reject_transfer_path(@channel)
```

Comment:
Would you use `token_reject_transfer_url(channel, channel.contest_token)` and keep the helper skinny?